app/policies/membership_policy.rb

```diff
@@ -27,7 +27,7 @@ def resolve(action)
     end
   end
 
-  scope :index, :show, :update, :destroy, :update_links, :destroy_links, with: Scope
+  scope :index, :show, :update, :destroy, with: Scope
```

Comment:
As I understand, this is going away because it doesn't support linking in the routes. It was listed in the can_through_parent call before, but as I've now found out that list was meaningless and not actually referenced by anything